README.rst

```diff
@@ -149,6 +149,28 @@ Advanced Usage
 
 See below for more options.
 
+Order Your Grouped Results with order_by and limit
```

Comment:
Could we move this to the analysis section? It will probably be more part of regularly querying than an "advanced" use case.

Reply:
Sure thing.